MAINTAINERS

```diff
@@ -1,2 +1,3 @@
+Nick Owens <nick.owens@coreos.com> (@mischief)
```

Comment:
can you either preserve the existing last name alphabetical order or just change to a,j,n ?